app/src/androidTest/java/com/pse2022/timefy/ui/dashboard/DashboardUiTest.kt

```diff
@@ -0,0 +1,196 @@
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.firebase.auth.GetTokenResult
+import com.pse2022.timefy.data.AuthManager
+import com.pse2022.timefy.data.api.EventService
+import com.pse2022.timefy.data.api.UserService
+import com.pse2022.timefy.ui.dashboard.viewModel.DashboardViewModel
+import com.pse2022.timefy.ui.theme.TimefyTheme
+import com.pse2022.timefy.ui.utils.TestTags
+import kotlinx.coroutines.flow.StateFlow
+import okhttp3.OkHttpClient
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import org.junit.*
+import org.junit.runner.RunWith
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+class MockAuthManager: AuthManager {
+    override var userService: UserService
+        get() = TODO("Not yet implemented")
+        set(value) {}
+    override val isAuthorized: StateFlow<Boolean>
+        get() = TODO("Not yet implemented")
+
+    override fun authorizeUser(onSuccess: () -> Unit, onError: () -> Unit) {
+        TODO("Not yet implemented")
+    }
+
+    override fun getUserToken(): GetTokenResult? {
+        TODO("Not yet implemented")
+    }
+
+    override fun getFullName(): String {
+        TODO("Not yet implemented")
+    }
+
+    override fun getFirstName(): String {
+        return "Max Mustermann"
+    }
+
+    override suspend fun syncUserWithServer() {
+        TODO("Not yet implemented")
+    }
+
+    override suspend fun signOut() {
+        TODO("Not yet implemented")
+    }
+
+}
+
+@RunWith(AndroidJUnit4::class)
+class DashboardUiTest {
+
+    @get:Rule
+    val composeRule = createComposeRule()
+
+    private val server = MockWebServer()
+    lateinit var authManager: AuthManager
+    lateinit var eventService: EventService
+    lateinit var viewModel: DashboardViewModel
+
+    @Before
+    fun setUp() {
+
+        server.start(8000)
+
+        val baseURL = server.url("/").toString()
+
+        val okHttpClient = OkHttpClient
+            .Builder()
+            .build()
+
+        eventService = Retrofit.Builder()
+            .baseUrl(baseURL)
+            .addConverterFactory(GsonConverterFactory.create())
+            .client(okHttpClient)
+            .build().create(EventService::class.java)
+
+        authManager = MockAuthManager()
+
+        composeRule.setContent {
+            TimefyTheme() {
+                viewModel = viewModel(factory = DashboardViewModel.provideFactory(eventService, authManager))
+                DashboardView(viewModel, null)
+            }
+        }
+    }
+
+    @After
+    fun tearDown() {
+        server.shutdown()
+    }
+
+    @Test
+    fun testNewlyPlannedEvents() {
+        server.enqueue(
+            MockResponse()
+                .setResponseCode(200)
+                .setBody(
+                    """
+                    [{
+		"id": "UUID",
+		"title": "Event title",
+		"duration": 60,
+		"synced": false,
+		"startDate": 0,
```

Comment:
```suggestion
		"startDate": 3600,
```
If we want to test the 01.01.1970 01:00 - 02:00, this has to be 3600

Reply:
Actually for me it is `0` because in our time zone `3600` equals to `Jan 01 1970 02:00:00 GMT+0100 (CET)`